clients/src/test/java/org/apache/kafka/common/requests/ConsumerGroupHeartbeatRequestTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.requests;
+
+import org.apache.kafka.common.message.ConsumerGroupHeartbeatRequestData;
+import org.apache.kafka.common.protocol.Errors;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ConsumerGroupHeartbeatRequestTest {
+
+    @Test
+    void testGetErrorConsumerGroupHeartbeatResponse() {
+        ConsumerGroupHeartbeatRequestData data = new ConsumerGroupHeartbeatRequestData();
+        ConsumerGroupHeartbeatRequest request = new ConsumerGroupHeartbeatRequest.Builder(data).build();
+        Throwable e = Errors.UNSUPPORTED_VERSION.exception();
+        int throttleTimeMs = 1000;
+        ConsumerGroupHeartbeatResponse response = request.getErrorResponse(throttleTimeMs, e);
+
+        assertEquals(throttleTimeMs, response.throttleTimeMs());
+        ApiError apiError = ApiError.fromThrowable(e);
```

Comment:
do we really need to involve the `ApiError` component here? seems we only use it to retrieve the code in the end, but that's in the `Errors` already. Could we simplify to just assert the response against the `error.code()` and `error.message`?